config/cscs.py

```diff
@@ -244,6 +284,29 @@ class ReframeSettings:
         },
 
         'environments': {
+
+            'ault': {
+                'PrgEnv-gnu': {
+                    'type': 'ProgEnvironment',
+                    'modules': ['gcc/8.3.0', 'cuda/10.1', 'openmpi/4.0.1'],
```

Comment:
I would just use the defaults so that one could easily do `-M gcc:gcc/10` when gcc 10 comes out and is installed.